packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -469,6 +469,27 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 		);
 	}
 
+	/**
+	 * Prints a message to the console.
+	 * @param message The message to print.
+	 */
+	async helloWorld(message: string) {
+		const phpResponse = await this.run({
+			code: `
+				<?php
+				$fullMessage = "PHP: ${message} \n";
+				echo $fullMessage;
+				`,
+		});
+
+		if (phpResponse.errors) {
```

Comment:
Yup! I wonder if we should just throw an exception instead. Thoughts on that? My worry is a scenario where PHP crashes without any output in stderr which could be confusing to the developer, although we could perhaps detect that based on the exit code and provide some useful guidelines in the error message.

Reply:
Would a promise reject be good here?

Reply:
Yup!